app/controllers/api/v1/contacts_controller.rb

```diff
@@ -4,9 +4,17 @@ module Api
   module V1
     class ContactsController < ApiController
       def index
-        program = current_user.program
+        render json: ContactService.new(program).as_json
+      end
+
+      private
+
+      def program
+        current_user && current_user.program
+      end
 
-        render json: program.try(:contacts) || []
+      def restrict_access
```

Comment:
@radinreth what is this method use for?

Reply:
I am overriding super class method,

because mobile user may or may not pass the token it the user is new user